src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
Per the checklist, let's return `Optional<MovieSession>` in `get()` method on the service layer, and the actual entity on the service layer. This will prevent `NullPointerException` and enforce good practice of handling the absence of a value.